xikolo-ios/ViewControllers/Courses/CourseContentTableViewController.swift

```diff
@@ -79,6 +79,19 @@ class CourseContentTableViewController: UITableViewController {
         }
     }
 
+    func showProctoringDialog() {
+        let title = NSLocalizedString("This item needs to be proctored", comment: "Shown in proctoring dialog")
+        let message = NSLocalizedString("You can't do this on your mobile device.", comment: "Shown in proctoring dialog")
```

Comment:
That's why I asked you to take care of it. The text should be more expressive. A good start point could be the qualified certificate page at our web client, to align the wording. For example I'm not sure if the user knows what proctoring means or if this is only an internal term.